src/dbnode/integration/cluster_add_one_node_test.go

```diff
@@ -258,4 +262,13 @@ func TestClusterAddOneNode(t *testing.T) {
 	for i := range setups {
 		verifySeriesMaps(t, setups[i], namesp.ID(), expectedSeriesMaps[i])
 	}
+
+	if verifyCommitlogCanBootstrapAfterNodeJoin {
+		// Verify that the node that joined the cluster can immediately bootstrap
+		// the data it streamed from its peers from the commit log as soon as
+		// the bootstrapping process completes.
+		require.NoError(t, setups[1].stopServer())
+		startServerWithNewInspection(t, opts, setups[1])
```

Comment:
do you need to reset the bootstrappers here - i.e. ensure it doesn't use peers again?

Reply:
nah startServerWithNewInspection completely reconstructs the bootstrappers